tools/md-formatter/form.js

```diff
@@ -0,0 +1,245 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable import/no-unresolved */
+import 'https://da.live/nx/public/sl/components.js';
+import getStyle from 'https://da.live/nx/utils/styles.js';
+import { LitElement, html } from 'https://da.live/nx/deps/lit/lit-core.min.js';
```

Comment:
Isn't this import mapped?
```suggestion
import { LitElement, html } from 'da-lit';
```

Reply:
It is mapped in the html, not in JS, added the import to our test config.